File: quality/tests/fixtures/strict_preset/loose_pass_strict_fail.py
# Loose passes, strict fails example (issue #615).
#
# Unsorted imports pass the loose selection (E4, E7, E9, F has no I001)
# and fail the strict preset (I). Illustrative only; not executed by the
# qualification harness as a tool run.
import sys
import os


def hello(name):
    return "hello " + name

File: quality/tests/fixtures/strict_preset/loose_pass_strict_fail.ts
// Loose passes, strict fails example.
//
// Implicit-any `name` passes tsc with strict false and fails with
// strict true (noImplicitAny); explicit `any` is the Biome strict
// `noExplicitAny` illustration. Illustrative only; not executed by the
// qualification harness as a tool run.
export function greet(name) {
	const result: any = name;
	return result;
}
